src/Sarif/Schemata/sarif-schema.json

```diff
@@ -560,6 +596,17 @@
                     }
                 },
 
+                "addresses": {
+                    "description": "The addresses associated with external property files, if any.",
```

Comment:
>The addresses associated with external property files, if any. [](start = 36, length = 62)

This s/be 

"An array of external property files containing run.addresses arrays to be merged with the root log file."